### PR TITLE
docs: Update demo projects list

### PR DESCRIPTION
**Pull request by raitisbe**

### CODE REVIEW

File: projects/hslayers/README.md [hunk 1]
```diff
@@ -13,10 +13,14 @@ HSLayers-NG is a library which extends OpenLayers 6 functionality by providing a
 
 ## Demo
 
-Check out the examples to get an idea:  
+Check out the examples and live projects to get an idea:  
 * http://ng.hslayers.org/examples/
-* http://opentransportnet.eu/create-maps
-* http://sdi4apps.eu/spoi
```
Comment (jmacura) [inline projects/hslayers/README.md:19]: I'd like to object that sdi4apps.eu/spoi (or https://app.hslayers.org/spoi/) is a live project ;-)

Comment (jmacura) [reply projects/hslayers/README.md:19]: But two active projects per library is probably enough, I admit.

Comment (raitisbe) [reply projects/hslayers/README.md:19]: > I'd like to object that sdi4apps.eu/spoi (or https://app.hslayers.org/spoi/) is a live project ;-)

You're right, I will add it back.

File: projects/hslayers/README.md [hunk 2]
```diff
@@ -13,10 +13,15 @@ HSLayers-NG is a library which extends OpenLayers 6 functionality by providing a
...
 ## Demo
 
-Check out the examples to get an idea:  
+Check out the examples and live projects to get an idea:  
 * http://ng.hslayers.org/examples/
-* http://opentransportnet.eu/create-maps
+* https://groundwater.smartagro.lv/fie/index.html
+* https://www.smartafrihub.com/cs/map1/
 * http://sdi4apps.eu/spoi
+* (Material) https://www.smartafrihub.com/cs/best-practises
```
Comment (fzadrazil) [inline projects/hslayers/README.md:21]: I would replace this link with https://atlasbestpractices.com, which opens the map directly.

Comment (raitisbe) [reply projects/hslayers/README.md:21]: Ok

**Pull request merged**